packages/service-library/src/servicelib/async_utils.py

```diff
@@ -205,3 +212,40 @@ async def worker(in_q: Queue[QueueElement], out_q: Queue) -> None:
         return wrapper
 
     return decorator
+
+
+def with_delay(
```

Comment:
MINOR: what about `delayed_start`?
`with_delay` does not specify whether the delay is before or after awaiting the coroutine

Reply:
ok